tools/file_packager.py

```diff
@@ -203,6 +203,16 @@
 var Module = typeof %(EXPORT_NAME)s !== 'undefined' ? %(EXPORT_NAME)s : {};
 ''' % {"EXPORT_NAME": export_name}
 
+ret += '''
+try {
+   Module.checkABI(1997);
+}
+catch (err)
+{
+  throw err;
```

Comment:
This "catch and throw again" is redundant.  I think you just need the line `Module.checkABI(1997)`.